formats/MrssFormat.php

```diff
@@ -60,12 +98,11 @@ public function stringify(){
 			$items .= <<<EOD
 
 	<item>
-		<title>{$itemTitle}</title>
-		<link>{$itemUri}</link>
-		<guid isPermaLink="true">{$itemUri}</guid>
-		<pubDate>{$itemTimestamp}</pubDate>
-		<description>{$itemContent}{$entryEnclosuresWarning}</description>
-		<author>{$itemAuthor}</author>
+		{$entryTitle}
+		{$entryLink}
+		<guid isPermaLink="false">{$entryID}</guid>
```

Comment:
Will `isPermalink="false"` cause issues for existing feeds (i.e. mark everything as unread)?
I'm not sure if it does, maybe the flag should be set to `true` if `$entryID` is equal to `$itemUri`?

Reply:
Well, the specification says:
> There are no rules for the syntax of a guid. Aggregators must view them as a string. It's up to the source of the feed to establish the uniqueness of the string.

> If the guid element has an attribute named isPermaLink with a value of true, the reader may assume that it is a permalink to the item, that is, a url that can be opened in a Web browser, that points to the full item described by the <item> element.

> isPermaLink is optional, its default value is true. If its value is false, the guid may not be assumed to be a url, or a url to anything in particular.

That means, it should not cause any issues on `isPermalink="false"`, the string is the same. However, if this is omitted or set to `true`, it must be a valid URL.

We _could_ set it to `true` if the guid is the same as `$itemUri`, it's just another condition. Do we want this?

Reply:
`isPermalink` is now set to `true` if it is the item URI.

Reply:
Thanks for taking care of it. :+1: